src/components/Footer.svelte

```diff
@@ -0,0 +1,147 @@
+<style>
+  footer {
+    padding-top: 50px;
+    padding-bottom: 50px;
+    background: #222;
+    color: #fff;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  .container-fluid {
+    max-width: 1200px;
+  }
+
+  img {
+    max-width: 100px;
+    margin-bottom: 40px;
+  }
+
+  .sitemap-title {
+    text-transform: uppercase;
+    font-weight: 700;
+    letter-spacing: 1px;
+  }
+
+  .sitemap-list {
+    margin: 13px 0;
+    padding: 0;
+    list-style: none;
+    color: #aaaaaa;
+  }
+
+  a {
```

Comment:
Remove this style:
- `cursor` is already set globally (We should remove it later on sine it is not needed).
- `text-decoration` should be lift to global styles.